IlmBase/IlmThread/IlmThreadPool.cpp

```diff
@@ -485,7 +489,7 @@ TaskGroup::Data::~Data ()
 
     isEmpty.wait ();
 
-#ifdef ENABLE_SEM_DTOR_WORKAROUND
+#if ENABLE_SEM_DTOR_WORKAROUND
```

Comment:
Is ENABLE_SEM_DTOR_WORKAROUND guaranteed to be always defined? The existing code assumes that it is not defined at all if the workaround is not desired. There is no guarantee that #if will result in a value of zero if it is not defined; this behavior varies between preprocessor implementations.

Reply:
I added one to the ENABLE_SEM_DTOR_WORKAROUND definition to make it even active in line 66.